docs/deploy-development-rp.md

```diff
@@ -175,6 +175,11 @@
        "https://localhost:8443/providers/Microsoft.RedHatOpenShift/operations?api-version=2020-04-30"
      ```
 
+   * View RP logs in a friendly format:
+
+     ```bash
+     journalctl _COMM=aro -o json --since "15 min ago" -f | jq '{msg: .MESSAGE, component: .COMPONENT} | select (.component != null and (.component | contains("access"))|not) | .msg'
```

Comment:
fwiw you can use `jq -r` to remove the quotes
also `'select (.COMPONENT // "" | contains("access") | not) | .MESSAGE'` is worth considering

Reply:
Good suggestions :) 
Updated